persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/message/MessageDAOImpl.kt

```diff
@@ -521,8 +346,204 @@ class MessageDAOImpl(
 
     override val platformExtensions: MessageExtensions = MessageExtensionsImpl(queries, mapper, coroutineContext)
 
-    private fun MessageEntityContent.Asset.hasValidRemoteData() =
-        assetId.isNotEmpty() && assetOtrKey.isNotEmpty() && assetSha256Key.isNotEmpty()
 }
 
+internal interface MessageInsertExtension {
```

Comment:
MessageInsertExtension not sure if the name is correct ? I do not see it really as an extension

Reply:
this class is meant only for inserting one message and to share logic between MigrationDAO and MessageDAO

Reply:
Whilst I couldn't help with any better name suggestion, I think adding a bit of context here in form of Kdoc would be nice.

Explaining that this is mainly used to share a bit of logic between `MessageDAO` and `MigrationDAO`